includes/class-wc-install.php

```diff
@@ -122,6 +122,9 @@ class WC_Install {
 		),
 		'3.5.3' => array(
 			'wc_update_353_db_version',
+		),
+		'1.0.0' => array(
+			'cc_update_100_default_base_color',
```

Comment:
I think the approach of using `1.0.0` here is going to cause problems later on because it's less than Woo versions that have already been released and installed. As an example, what happens when CC gets to `2.0.0`? There is already a `2.0.0` key in this array. There may also be more subtle issues if this version number is compared for `>=` equality anywhere else in the code.

There should also be a `cc_update_100_db_version` function here, but the exact behavior of that function depends on what approach is chosen for these migrations. A couple of example approaches to consider:

- Use something like `3.5.3+cc1.0.0` in this field
- Introduce a new "db_version" field for CC and a new array of upgrade/migration functions